export/mlab_export_test.py

```diff
@@ -0,0 +1,492 @@
+    
+    last_time = mlab_export.LastTime(fake_tstamp)
+    last_time.open()
+    return_lock = last_time.lock()  # Should fail.
+    last_time.close()
+
+    self.assertFalse(return_lock)
+
+  @mock.patch('mlab_export.logging.error')
+  @mock.patch('mlab_export.LastTime.lock')
+  def testcover_main_exit(self, mock_last_time_lock, mock_error):
+    fake_tstamp = os.path.join(self._testdata_dir, 'fake.tstamp')
+    mlab_export.LAST_EXPORT_FILENAME = fake_tstamp
+    mock_last_time_lock.return_value = False
+
+    self.assertRaises(SystemExit, mlab_export.main)
+
+    self.assertTrue(mock_last_time_lock.called)
+    self.assertTrue(mock_error.called)
+
+  @mock.patch('mlab_export.parse_args')
+  @mock.patch('mlab_export.rrd_list')
+  def testcover_main_list(self, mock_rrd_list, mock_parse_args):
+    fake_tstamp = os.path.join(self._testdata_dir, 'fake.tstamp')
+    mlab_export.LAST_EXPORT_FILENAME = fake_tstamp
+    mock_options = disable_show_options(mock.Mock())
+    mock_options.show_nagios = True  # Enable one show option.
+    mock_parse_args.return_value = mock_options
+
+    mlab_export.main()
+
+    self.assertTrue(mock_parse_args.called)
+    self.assertTrue(mock_rrd_list.called)
+
+  @mock.patch('mlab_export.parse_args')
+  @mock.patch('mlab_export.rrd_export')
+  def testcover_main_export(self, mock_rrd_export, mock_parse_args):
+    fake_tstamp = os.path.join(self._testdata_dir, 'fake.tstamp')
+    mlab_export.LAST_EXPORT_FILENAME = fake_tstamp
+    mock_options = disable_show_options(mock.Mock())
+    mock_options.update = True
+    mock_options.rrddir_prefix = mlab_export.RRD_PREFIX
+    mock_options.ts_start = time.time() - 60*60
+    mock_options.ts_end = time.time()
+    mock_parse_args.return_value = mock_options
+
+    mlab_export.main()
+
+    self.assertTrue(mock_parse_args.called)
+    self.assertTrue(mock_rrd_export.called)
+
+  def testcover_init_global(self):
+    mlab_export.init_global()
+
+  @mock.patch('mlab_export.logging.error')
+  def testcover_parse_args(self, mock_error):
+    mlab_export.sys.argv = [
+        'mlab_export.py', '--bad-argument', '--causes-error']
+
+    self.assertRaises(SystemExit, mlab_export.parse_args, 0)
+
+    self.assertTrue(mock_error.called)
+
+  def testunit_align_timestamp(self):
+    fake_ts = 1410341339
+    expected_6 = fake_ts - 5
+    expected_10 = fake_ts - 9
+    expected_30 = fake_ts - 29
+
+    ts_6 = mlab_export.align_timestamp(fake_ts, 6)
+    ts_10 = mlab_export.align_timestamp(fake_ts, 10)
+    ts_30 = mlab_export.align_timestamp(fake_ts, 30)
+
+    self.assertEqual(ts_6, expected_6)
+    self.assertEqual(ts_10, expected_10)
+    self.assertEqual(ts_30, expected_30)
+
+  @mock.patch('mlab_export.time.time')
+  def testunit_default_end_time(self, mock_time):
+    mock_time.return_value = 1401401409
+    mock_options = mock.Mock()
+    mock_options.length = 3600
+    mock_options.step = 10
+    mock_options.update = False
+    expected_ts_end = 1401400800
```

Comment:
Same note here. Can we use numbers that are easy to verify mentally? Like changing length to `5000`